presto-mongodb/src/main/java/com/facebook/presto/mongodb/MongoMetadata.java

```diff
@@ -62,6 +65,10 @@ public class MongoMetadata
 
     private final AtomicReference<Runnable> rollbackAction = new AtomicReference<>();
 
+    private Constraint<ColumnHandle> constraint;
+
+    private static final String DELETE_ROW_ID = "_presto_artificial_column_handle_for_delete_row_id_";
```

Comment:
Can you explain a bit how this DELETE_ROW_ID is used?  I am not familiar with the code here

Reply:
In MongoDB, we don't have the concept of row IDs, and all the data is stored as documents. 

This is an artificial rowID that helps to mimic the concept of rowID on such DBs, so that presto can understand each document objects as individual rows.